store/mockstore/mocktikv/cop_handler_dag.go

```diff
@@ -471,29 +468,33 @@ func (mock *mockCopStreamClient) Recv() (*coprocessor.Response, error) {
 		return &resp, nil
 	}
 	streamResponse := tipb.StreamResponse{
-		Error:        toPBError(err),
-		EncodeType:   tipb.EncodeType_TypeDefault,
-		Data:         data,
-		OutputCounts: counts,
-	}
+		Error:      toPBError(err),
+		EncodeType: tipb.EncodeType_TypeDefault,
+		Data:       data,
+	}
+	// The counts was the output count of each executor, but now it is the scan count of each range,
+	// so we need a flag to tell them apart.
+	streamResponse.OutputCounts = make([]int64, 1+len(counts))
```

Comment:
How about `streamResponse.OutputCounts = append(counts, -1)`?

Reply:
We cannot do this because the future streaming result may still use the `count`.